content/blog/2018-03-23-hydroscoper.md

```diff
@@ -0,0 +1,157 @@
+---
+slug: "hydroscoper"
+title: ὕδωρ + σκοπῶ
+package_version: 1.0.0
+authors:
+  - name: Konstantinos Vantas
+    url: https://kvantas.github.io/
+date: 2018-03-23
+categories: blog
+topicid: 666666666
+tags:
+  - r
+  - community
+  - software
+  - review
+  - package
+  - data-access
+  - hydrology
+  - Greece
+  - hydroscope
+output:
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+> Hydrology is a concept to unify statistics, data analysis and numerical models in order to understand and analyze the endless circulation of water between the earth and its atmosphere.
+
+That's a lot alike Data Science, isn't it? Hydrologic Processes evolve in space and time, are extreme complex and we may never comprehend them. For this reason Hydrologists use models where their inputs and outputs are measurable variables: climatic and hydrologic data, land uses, vegetation coverage, soil type etc.  
+
+Not so many years ago, I used to struggle for these data: travelling 500 km away searching in underground storages, requesting from peers and institutes or even use sophistries to get them. Nowadays, hydrologic data come from a variety of sources such as satellites, online weather stations, national climatic databases, etc. But problems never end: these data have a variety of types and formats, errors, noise and a big amount of missing values. 
+
+In Greece, the Hydroscope (ὕδωρ + σκοπῶ) project started in the 1990s, aimed to organize data belonging to various institutions and to create a database of hydrometeorological information. Today this database, [Hydroscope](http://www.hydroscope.gr/), is available online. Unfortunately, all the data in Hydroscope are in Greek and there are no convenient look-up tables. 
+
+## The hydroscoper package
```

Comment:
This heading looks quite large. Maybe try one level smaller `### The hydroscoper package`